Gemfile

```diff
@@ -76,3 +78,5 @@ gem 'rails-assets-jquery-hoverintent', source: 'https://rails-assets.org'
 # Rubocop
 gem 'rubocop'
 gem 'rubocop-rspec'
+gem 'ensurance'             # better way to ensure you're dealing with a Model
```

Comment:
As much as I like your gem, I am always afraid that maintainer will someday stop maintaining their gems. That's why I don't like adding a gem for everything. We should keep the amount of external gems very low. Maybe move the `ensure` method into a lib instead of using an external gem?

Reply:
totally understand.  it's very small and can easily be added as a lib.

Reply:
however -- it *is* open source with an MIT license. 
If I got hit by a bus, or stopped maintaining it, you'd still have the source available.   Which is no different than copy/pasting the source into your own project, *except* that you don't get any fixes that others create.

Reply:
That is true. Okay, lets keep it then.